packages/api-client/src/views/Request/libs/request.ts

```diff
@@ -10,11 +11,63 @@ import type { RequestExampleParameter } from '@scalar/oas-utils/entities/spec'
  * - maximum
  */
 export const hasItemProperties = (item: RequestExampleParameter) =>
-  Boolean(
-    item.description ||
-      item.type ||
-      item.default ||
-      item.format ||
-      item.minimum ||
-      item.maximum,
-  )
+  Boolean(item.description || item.type || item.default || item.format || item.minimum || item.maximum)
+
+/**
+ * Checks if the value of a RequestExampleParameter is the expected type or format
+ * Returns an alert message if the value is not in the correct type or format, otherwise false
+ */
+export const parameterIsInvalid = (item: RequestExampleParameter) => {
+  return computed(() => {
```

Comment:
why is this computed? Cant we just use it without the computed? I think this will be slower unless computed caches by input and not by instance 🤔